src/js/MenuItem.js

```diff
@@ -0,0 +1,164 @@
+//MenuItem.js
+var Glyphicon = require('react-bootstrap/lib/Glyphicon.js');
+var Well = require('react-bootstrap/lib/Well.js');
+
+
+var MenuItem = React.createClass({
+  getInitialState: function(){
+      return {
+        flagged: false,
+        wrong: false,
+        needSwitch: false,
+        isCorrect: false,
+        displayWrong: true,
+        displayNeedSwitch: true,
+        displayIsCorrect: true
+      };
+  },
```

Comment:
I think, starting from this point, your indentation is off. `toggleFlag`, `toggleWrong`, etc. should be indented like `getInitialState`.